ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -73,6 +142,88 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
     static get is() {
       return 'nuxeo-checkbox-aggregation';
     }
+
+    static get properties() {
+      return {
+        _buckets: {
+          type: Array,
+          value: () => [],
+        },
+        /**
+         * Make the element collapsible
+         */
+        collapsible: {
+          type: Boolean,
+          value: false,
+        },
+
+        /**
+         * Label of the element
+         */
+        label: {
+          type: String,
+          value: '',
+        },
+
+        _show: {
+          type: Boolean,
+          value: false,
+        },
+
+        _slicedBuckets: {
+          type: Array,
+          value: () => [],
+        },
+
+        _visibleBuckets: {
+          type: Array,
+          value: () => [],
+        },
+
+        /**
+         * If collapsible, the default visible items.
+         */
+        visibleItems: {
+          type: Number,
+          value: 8,
+        },
+      };
+    }
+
+    _checkCollapse(buckets) {
+      if (!this.collapsible && buckets.length) {
+        this._visibleBuckets = buckets;
+        return true;
+      }
+      return false;
+    }
+
+    _setBuckets(buckets) {
+      if (this.collapsible && buckets.length) {
+        this._buckets = buckets;
```

Comment:
`this._buckets` will always be the same as `this.buckets`, right?
Probably it is also something we can drop 🙂